### ISSUE TITLE
All plugins get deactivated after running a check of a plugin

### ISSUE THREAD

**Issue by wp-seopress** [10up/plugin-check#545]
Hi,

Not sure how it happens but sometimes when we run a check on a plugin, all the active plugins get deactivated except Plugin Check.

**Comment by swissspidy** [comment]
I cannot reproduce this at first glance. It would also be surprising to me, since runtime checks spin up a new database table for activating just the plugin being tested. Your actual site is unaffected by that.

**Comment by davidperezgar** [comment]
Could you give more context? Do you reproduce that problem more times?

**Comment by wp-seopress** [comment]
I have got this issue again.

Error in debug.log file:

```
PHP Fatal error:  Uncaught Exception: Check with the slug "enqueued_scripts_scope" does not exist. in /app/public/wp-content/plugins/plugin-check/includes/Checker/Default_Check_Collection.php:158
Stack trace:
#0 /app/public/wp-content/plugins/plugin-check/includes/Checker/Abstract_Check_Runner.php(413): WordPress\Plugin_Check\Checker\Default_Check_Collection->require(Array)
#1 /app/public/wp-content/plugins/plugin-check/includes/Checker/Abstract_Check_Runner.php(284): WordPress\Plugin_Check\Checker\Abstract_Check_Runner->get_checks_to_run()
#2 /app/public/wp-content/plugins/plugin-check/includes/Utilities/Plugin_Request_Utility.php(101): WordPress\Plugin_Check\Checker\Abstract_Check_Runner->prepare()
#3 /app/public/wp-includes/class-wp-hook.php(324): WordPress\Plugin_Check\Utilities\Plugin_Request_Utility::WordPress\Plugin_Check\Utilities\{closure}('')
#4 /app/public/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#5 /app/public/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /app/public/wp-settings.php(487): do_action('muplugins_loade...')
#7 /app/public/wp-config.php(92): require_once('/Users/benjamin...')
#8 /app/public/wp-load.php(50): require_once('/Users/benjamin...')
#9 /app/public/wp-admin/admin-ajax.php(22): require_once('/Users/benjamin...')
#10 {main}
  thrown in /app/public/wp-content/plugins/plugin-check/includes/Checker/Default_Check_Collection.php on line 158

**Comment by wp-seopress** [comment]
Not sure if it's important: I scan a development version of a plugin (with a lot of hidden files, config files, deployment files etc).

**Comment by ernilambar** [comment]
@wp-seopress Can you please confirm the Plugin Check is updated to latest? I think missing `enqueued_scripts_scope` was fixed in recent PRs.

**Comment by wp-seopress** [comment]
version 1.0.2

**Comment by ernilambar** [comment]
Hmm, that issue was fixed in https://github.com/WordPress/plugin-check/commit/af7f8297d53a7c05d892d2964bc54ab1ae9f1013 and was shipped in `1.0.2`. Not sure what is the issue then.

**Comment by swissspidy** [comment]
Closing as this can‘t be reproduced. Feel free to reopen with more details if this is still an issue for you
